src/vs/workbench/contrib/scm/browser/scmViewPane.ts

```diff
@@ -1788,9 +1788,8 @@ class SCMInputWidget {
 		}
 
 		const uri = URI.from({
-			scheme: Schemas.vscode,
-			authority: 'scm',
-			path: `/${input.repository.provider.contextValue}/${input.repository.provider.id}/input`,
+			scheme: Schemas.vscodeSourceControl,
+			path: `scm/${input.repository.provider.contextValue}/${input.repository.provider.id}/input`,
```

Comment:
Do we still need `scm/` in the path?

Does this now require another adoption change in GHPR?

Reply:
I like that. I will remove the `scm` segment and will work with @alexr00 to make sure that the GHPR extension is updated.